components/unitStats/unitTable.tsx

```diff
@@ -239,7 +248,27 @@ const getCellVisual = (colSetup: tableColSetup, unit: CustomizableUnit) => {
       />
     );
 
-  if (!colSetup.customVisual) return (unit as any)[colSetup.key];
+  if (!colSetup.customVisual) {
+    if (colSetup.key === "screen_name") {
+      return (
+        <Tooltip label={(unit as any)[colSetup.key]}>
+          <Text sx={{ textOverflow: "ellipsis", overflow: "hidden" }}>
+            {(unit as any)[colSetup.key]}
+          </Text>
+        </Tooltip>
+      );
+    }
+    return (
+      <Anchor
+        color="undefined"
+        underline={false}
+        component={Link}
+        href={`/explorer/races/${unit.faction}/units/${unit.id}`}
```

Comment:
Can we get this url from here https://github.com/cohstats/coh3-stats/blob/master/src/routes.ts 

if it's not there, please add it there 👍  thank you